patterns/03-organisms/pageheaders/pageheader-dots/pageheader-dots.css

```
@@ -43,8 +27,7 @@
 }
 
 .pageheader-text-container {
```

At some point we should align the classnames of the component with the component name
```suggestion
.pageheader-dots__text-container{
```
@emilankersen  @korfitz @magnify